src/localization/en.json

```diff
@@ -18,16 +18,53 @@
     "openUserMenu": "Open user menu",
     "logoAlt": "Metatavu logo"
   },
+  "error": {
+    "totalTimeFetch": "Retrieving total time data has failed",
+    "totalTimeNotFound": "Could not find total time data",
+    "dailyEntriesFetch": "Retrieving daily entries has failed",
+    "dailyEntriesNotFound": "Could not find daily entries",
+    "admin": "A",
+    "changeLanguage": "A",
+    "openSettings": "A",
+    "openUserMenu": "A",
+    "logoAlt": "A",
```

Comment:
These seem like strange localizations?

Reply:
These were placeholder localizations to get rid of errors after merging. Forgot about them. Oops!